mp/src/game/shared/neo/neo_version.cpp

```diff
@@ -0,0 +1,34 @@
+#include "neo_version_info.h"
+
+#include "convar.h"
+#include "dbg.h"
+
+namespace {
+void neoVersionCallback()
+{
+#if defined(GAME_DLL)
+	static constexpr char HEADER[] = "neo_sv_version (Server's build info):";
```

Comment:
This should be updated to match `neo_version` probably

Reply:
done